app/src/main/res/layout-land/topic_practice_footer_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+
+    <variable
+      name="isSubmitButtonActive"
+      type="Boolean" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.topic.practice.practiceitemviewmodel.TopicPracticeFooterViewModel"/>
+  </data>
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
The indentation in this file is incorrect.
Press `Control + Alt + L` to fix indentation or check other files to fix that.

Reply:
This file exactly matches its portrait mode copy in terms of indentation. Should I proceed with  `Control+Alt+L` ?

Reply:
Yes, and fix that in portrait mode also.

Reply:
Done PTAL.